kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.testing.collection;
+
+
+import java.util.ArrayList;
+
+public class OddNumbersExterminator {
+
+   private ArrayList<Integer> evenNumbers = new ArrayList<>();
```

Comment:
to powinna byc zmienna lokalna, nie klasowa (inaczej evenNumer bedzie puchlo przy kazdym wywolaniu)